test/unit/traversing.js

```diff
@@ -743,6 +743,49 @@ QUnit.test( "contents()", function( assert ) {
 	assert.equal( c[ 0 ].nodeValue, "hi", "Check node,textnode,comment contents is just the one from span" );
 } );
 
+QUnit.test( "contents() for <template />", function( assert ) {
+	assert.expect( 6 );
+
+    jQuery( "body" ).append(
```

Comment:
If I append the HTML to `#qunit-fixture`, should I remove them after my testing?